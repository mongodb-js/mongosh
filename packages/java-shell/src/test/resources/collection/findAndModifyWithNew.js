// before
db.coll.deleteMany({});
db.coll.insertMany([{v: "a", number: 1}, {v: "b", number: 1}]);
// command
db.coll.findAndModify({query: {"v": "a"}, update: {$inc: {number: 1}}, new: true});
// command
db.coll.find();
// clear
db.coll.drop();
