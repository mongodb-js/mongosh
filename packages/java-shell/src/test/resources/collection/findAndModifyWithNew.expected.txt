{ "_id": <ObjectID>, "v": "a", "number": 2 }
[ { "_id": <ObjectID>, "v": "a", "number": 2 }, { "_id": <ObjectID>, "v": "b", "number": 1 } ]